### ISSUE TITLE
simply-reactive support

### ISSUE THREAD

**Issue by vasilev-gregory** [vadimkorr/svelte-carousel#129]
```
 TypeError: Cannot read properties of undefined (reading 'default')                                                                                                             
     at Module.createCarousel [as default] (/node_modules/svelte-carousel/src/components/Carousel/createCarousel.js:23:19)                                                      
     at Carousel.svelte:33:70                                                                                                                                                   
     at Object.$$render (/node_modules/svelte/internal/index.mjs:1892:22)                                                                                                       
     at eval (/src/routes/+page.svelte:137:88)                                                                                                                                  
     at Object.$$render (/node_modules/svelte/internal/index.mjs:1892:22)                                                                                                       
     at Object.default (root.svelte:41:38)                                                                                                                                      
     at eval (/node_modules/@sveltejs/kit/src/runtime/components/layout.svelte:8:41)                                                                                            
     at Object.$$render (/node_modules/svelte/internal/index.mjs:1892:22)                                                                                                       
     at root.svelte:40:37                                                                                                                                                       
     at $$render (/node_modules/svelte/internal/index.mjs:1892:22) 
```

```
  1 import simplyReactive from 'simply-reactive'
  2 
  3 import { NEXT, PREV } from '../../direction'
  4 import {
  5   getCurrentPageIndexByCurrentParticleIndex,
  6   getPartialPageSize,
  7   getPagesCountByParticlesCount,
  8   getParticleIndexByPageIndex,
  9 } from '../../utils/page'
 10 import { getClonesCount } from '../../utils/clones'
 11 import { getAdjacentIndexes } from '../../utils/lazy'
 12 import { getValueInRange } from '../../utils/math'
 13 import { get, switcher } from '../../utils/object'
 14 import { ProgressManager } from '../../utils/ProgressManager'
 15 
 16 function createCarousel(onChange) {
 17   const progressManager = new ProgressManager({
 18     onProgressValueChange: (value) => {
 19       onChange('progressValue', 1 - value)
 20     },
 21   })
 22 
 23   const reactive = simplyReactive(
```

**Comment by Kaikeplatinium** [comment]
I got the same problem, i tried the sveltekit support for usage and when run dev it works normally, but when dev build it crashes the same way

**Comment by Kaikeplatinium** [comment]
I think i found a fix, i install separatly `npm install simply-reactive`

**Comment by vadimkorr** [comment]
Fixed in `svelte-carousel@1.0.23`
